selfdrive/car/ford/values.py

```diff
@@ -122,10 +133,70 @@ def init_make(self, CP: car.CarParams):
       bus=0,
       auxiliary=True,
     ),
+    # Ecu.abs: Wheel Base (response[0] & 0xF0), Payload (response[0] & 0xF),
+    #          Steering Gear (response[1] & 0b11), Cruise Control Mode (response[5] & 0xF)
+    # Ecu.eps: Lane Keeping Aid (LKA) (response[6] & 0xFF), Traffic Jam Assist (TJA) (response[7] & 0xFF),
+    #          Lane Centering Assist (LCA) (response[8] & 0xFF)
+    Request(
+      [StdQueries.TESTER_PRESENT_REQUEST, ford_asbuilt_block_request(1)],
+      [StdQueries.TESTER_PRESENT_RESPONSE, ford_asbuilt_block_response(1)],
+      whitelist_ecus=[Ecu.abs, Ecu.eps],
+      logging=True,
+    ),
```

Comment:
This would significantly increase fingerprinting time for CAN fingerprinting cars or where Ford would be sorted first, perhaps we have a list of requests that only run if we fingerprint to this brand, and the responses are nicely given to the port in a dictionary style manner.

If we had no CAN fingerprinting cars, it does seem strange to do this in the fingerprinting stage anyway, blocking controlsd startup. We'll have to think about this a bit more.

Reply:
if we're just trying to figure out which of these are necessary, i'm fine shipping this to master for a while then trimming it down after some data comes in

Reply:
@incognitojam let's do that and figure out the refactor later

Reply:
Ok I've reverted the other changes and fixed the request block numbers